src/models/participant/participant.model.ts

```diff
@@ -0,0 +1,62 @@
+import mongoose from "mongoose";
+import {
+    participant as ParticipantType,
+    updateParticipant as updateParticipantType,
+} from "../../types/participant";
+import Participant from "./participant.schema";
+import ErrorWithStatusCode from "../../util/classes/ErrorWithStatusCode";
+
+export async function dbGetAllParticipants() {
+    const participants = await Participant.find();
+    if (!participants) {
+        throw new ErrorWithStatusCode(
+            "Error while getting all participants",
+            500
+        );
+    }
+    return participants;
+}
+export async function dbAddParticipant(participant: ParticipantType|updateParticipantType) {
+    const newParticipant = new Participant (participant);
+    if (!newParticipant) {
+        throw new ErrorWithStatusCode("Error while adding participant", 500);
+    }
+    
+    return await newParticipant.save();
+}
+
+export async function dbDeleteParticipantByEmail(email: string) {
+    let filter = {email:email};
```

Comment:
```suggestion
    let filter = {email};
```
When you already have a variable with the same name as a prop you want to put in an object, this is a short cut

Reply:
readability